djpressor/static/djpressor/js/form/form-image-upload-field.js

```diff
@@ -381,9 +381,10 @@ var djpressor = function () {
       };
 
       // disable form submission until stuff get uploaded
-      manager.isUploadInProgress = true;
+      manager.canSubmitForm = false;
```

Comment:
Is button state always depended from this flag? If so Let's make unified method

manager.canSubmitForm = false;
manager.changeSubmitBtnState(false);